lib/msf/core/post/windows/shadowcopy.rb

```diff
@@ -182,6 +182,25 @@ def start_vss
         return false
       end
     end
+    unless start_swprv
+      return false
+    end
+    return true
+  end
+
+  def start_swprv
+    vss_state = wmic_query('Service where(name="swprv") get state')
```

Comment:
why is that better than using WMI to do it? service_info also involves 4 calls across the wire instead of just one